### DISCUSSION TITLE
Still active Project?

### DISCUSSION THREAD

**Discussion by jmorris644** [SEPIA-Framework/sepia-docs#244] [General]
It looks like the last updates are over a year old. Is the development effort of this project still active?

**Comment by fquirin** [comment]
As you might have noticed in dec 2023 already, activity was rather low for the last ~2 years 🙈 . It became a lot of work to keep all the parts alive at the same time (server, clients, STT/TTS/wake-word integrations, Android app etc.) and with the emerging LLM technology, I thought it was time to take a little break and to find out where this project is going next. There has been some work on the STT server in the meantime (mostly regarding Whisper) and I'm constantly experimenting with local LLMs at the moment. We'll see what the future brings :-)

**Reply by RMac67** [reply]
I am one of thousands of techs and regular people who hate the LLM's for so many reasons, mostly because of the lack of security. I just came across SEPIA while looking for a a voice personal assistant that was off-line and functioned on my PC as well as my phone. If you would like help spreading the word, I can help with that.
